src/main/java/kr/codesqaud/cafe/domain/Qna.java

```diff
@@ -0,0 +1,47 @@
+package kr.codesqaud.cafe.domain;
+
+public class Qna {
```

Comment:
```suggestion
public class Article {
```
클래스 이름은 Question 이나 Article 같은 이름이 더 좋을 것 같습니다.